supermarq-benchmarks/supermarq/qcvv/irb.py

```diff
@@ -21,15 +21,248 @@
+            xs=np.array([[True], [True]], dtype=np.dtype("bool")),
+            zs=np.array([[False], [True]], dtype=np.dtype("bool")),
+            initial_state=0,
+        )
+    ),
+    # X**0.5, Y**0.5, X**0.5
+    cirq.CliffordGate.from_clifford_tableau(
+        cirq.CliffordTableau(
+            1,
+            rs=np.array([False, True], dtype=np.dtype("bool")),
+            xs=np.array([[True], [False]], dtype=np.dtype("bool")),
+            zs=np.array([[True], [True]], dtype=np.dtype("bool")),
+            initial_state=0,
+        )
+    ),
+    # Y**-0.5
+    cirq.CliffordGate.from_clifford_tableau(
+        cirq.CliffordTableau(
+            1,
+            rs=np.array([False, True], dtype=np.dtype("bool")),
+            xs=np.array([[False], [True]], dtype=np.dtype("bool")),
+            zs=np.array([[True], [False]], dtype=np.dtype("bool")),
+            initial_state=0,
+        )
+    ),
+]
+
+
+_S1_Y = [
+    # Y**0.5
+    cirq.CliffordGate.from_clifford_tableau(
+        cirq.CliffordTableau(
+            1,
+            rs=np.array([True, False], dtype=np.dtype("bool")),
+            xs=np.array([[False], [True]], dtype=np.dtype("bool")),
+            zs=np.array([[True], [False]], dtype=np.dtype("bool")),
+            initial_state=0,
+        )
+    ),
+    # X**-0.5 Y**-0.5 X**0.5
+    cirq.CliffordGate.from_clifford_tableau(
+        cirq.CliffordTableau(
+            1,
+            rs=np.array([True, False], dtype=np.dtype("bool")),
+            xs=np.array([[True], [False]], dtype=np.dtype("bool")),
+            zs=np.array([[True], [True]], dtype=np.dtype("bool")),
+            initial_state=0,
+        )
+    ),
+    # Y X**0.5
+    cirq.CliffordGate.from_clifford_tableau(
+        cirq.CliffordTableau(
+            1,
+            rs=np.array([True, False], dtype=np.dtype("bool")),
+            xs=np.array([[True], [True]], dtype=np.dtype("bool")),
+            zs=np.array([[False], [True]], dtype=np.dtype("bool")),
+            initial_state=0,
+        )
+    ),
+]
+
+
+def random_single_qubit_clifford() -> cirq.SingleQubitCliffordGate:
+    """Choose a random single qubit clifford gate.
+
+    Returns:
+        The random clifford gate.
+    """
+    Id = cirq.SingleQubitCliffordGate.I
+    H = cirq.SingleQubitCliffordGate.H
+    S = cirq.SingleQubitCliffordGate.Z_sqrt
+    X = cirq.SingleQubitCliffordGate.X
+    Y = cirq.SingleQubitCliffordGate.Y
+    Z = cirq.SingleQubitCliffordGate.Z
+
+    set_A = [
+        Id,
+        S,
+        H,
+        _reduce_single_qubit_clifford_seq([H, S]),
+        _reduce_single_qubit_clifford_seq([S, H]),
+        _reduce_single_qubit_clifford_seq([H, S, H]),
+    ]
+
+    set_B = [Id, X, Y, Z]
+
+    return _reduce_single_qubit_clifford_seq([random.choice(set_A), random.choice(set_B)])
+
+
+def random_two_qubit_clifford() -> cirq.CliffordGate:
+    """Choose a random two qubit clifford gate.
+
+    Returns:
+        The random clifford gate.
+    """
```

Comment:
Could be nice to link https://arxiv.org/abs/1402.4848 and/or add a few lines describing to save folks a trip to Cirq (although the link to Cirq's `qubit_characterizations.py` file above is also nice).